### DISCUSSION TITLE
CKeditor 4.17.1 giving 403 error if the html contains opening parenthesis

### DISCUSSION THREAD

**Discussion by mohdaftab** [livewire/livewire#5624] [1. Help]
Hello,

I am using CKeditor 4.17.1 with Livewire and I was getting 403 forbidden access when I copy/paste html code in the CKEditor and after a lot of testing I found out that it is only the opening parentheses that is causing the trouble.
It works great in my local server but not on shared hosting account from namecheap.

Please have a look at the screenshots below
![ss1](https://user-images.githubusercontent.com/25055576/227073092-ec00989b-7012-4c1e-be23-751b608001bb.jpg)
![ss2](https://user-images.githubusercontent.com/25055576/227073107-cc9ffaad-1fa6-446d-bd1a-585847e8d657.jpg)
![ss3](https://user-images.githubusercontent.com/25055576/227073372-adf0e1e8-3688-41fd-a2cc-59cce28cb01a.JPG)

Please let me know how do I fix this.

Thank you so much.

**Comment by joshhanley** [comment]
This issue has been converted to a discussion as we are now using discussions to track bugs and ideas.
See this announcement for more details https://github.com/livewire/livewire/discussions/3147

**Comment by winterant** [comment]
Have you found a way?

I have encountered a similar problem. I hava an idea that encoded form data in base64 before they are send to server. The Server would be no reason to intercept data encoded in base64.

But, I didn't found any way to implement the code using livewire.
